guides/common/modules/proc_performing-post-upgrade-tasks.adoc

```diff
@@ -10,13 +10,22 @@ This information allows Katello to display manifest labels, annotations, and inf
 As a result, migrations must be performed to pull this content from manifests into the database.
 
 ifdef::katello[]
-This migration takes time, so if you depend on container content and need minimal upgrade downtime, run the following command on {ProjectServer} for a pre-migration:
-
+.Procedure
+. This migration takes time, so if you depend on container content and need minimal upgrade downtime, run the following commands on {ProjectServer} for a pre-migration:
++
 [options="nowrap" subs="attributes"]
 ----
 # foreman-maintain advanced procedure run pulpcore-container-handle-image-metadata
 ----
 This command migrates data in the background without any need for downtime and reduces future upgrade downtime.
+
+. You will also need to run this command:
```

Comment:
@ianballou Can you elaborate on this?

Reply:
The commands must run in the order:
1) `pulpcore-container-handle-image-metadata`
2) `import_container_manifest_labels`

They have to be run sequentially. The commands only need to be run one time together.

Reply:
Then why isn't the rake task part of the maintain procedure?

Reply:
It was originally, but it was taken out because it should only run on smart proxies, not Foreman itself. Perhaps it could've still lived in there but only ran if the system was a smart proxy.  https://github.com/theforeman/foreman_maintain/pull/877/files

Reply:
> It was originally, but it was taken out because it should only run on Katello itself, not smart proxies. Perhaps it could've still lived in there but only ran if the system was a Katello server. https://github.com/theforeman/foreman_maintain/pull/877/files

foreman-maintain can constrain it only to be run if katello is present.

Reply:
Sorry for the partial reply, but I now see it was introduced as a separate procedure. So if anything this PR should call `foreman-maintain` to do it and let that run `foreman-rake`. We should treat `foreman-rake` as an advanced feature that we don't steer users to.

That also makes me think more about this: do we need to run the `ContainerHandleImageMetadata` procedure on all machines with Pulp? If so, does it mean you also need to run `IndexKatelloRepositoriesContainerMetatdata` after you run `ContainerHandleImageMetadata` on a content proxy?

Reply:
All machines indeed need both the rake script and the pulp-container migration run.

> So if anything this PR should call `foreman-maintain` to do it and let that run `foreman-rake`.

I looked into this, and for some reason the `IndexKatelloRepositoriesContainerMetatdata` procedure was not available in the list of advanced procedures to run, unlike the Pulp container one.  Since the release is already out in the wild, I figured we should document what's actually possible.